src/mono/mono/mini/exceptions-amd64.c

```diff
@@ -1044,14 +1044,14 @@ mono_arch_unwindinfo_add_push_nonvol (PUNWIND_INFO unwindinfo, MonoUnwindOp *unw
 
 	codeindex = MONO_MAX_UNWIND_CODES - (++unwindinfo->CountOfCodes);
 	unwindcode = &unwindinfo->UnwindCode [codeindex];
-	unwindcode->UnwindOp = UWOP_PUSH_NONVOL;
```

Comment:
These are unrelated.

Reply:
Yes, this is fixing a build error on Windows that I fixed as part of this PR (since I implemented the change on Windows, so it was in the way). If it make sense it could be moved into a separate PR or kept in this one, for me it doesn't matter.